starport/cmd/account.go

```diff
@@ -116,3 +116,15 @@ func getPassphrase(cmd *cobra.Command) (string, error) {
 
 	return pass, nil
 }
+
+func checkAccountExist(cmd *cobra.Command, name string) error {
+	ca, err := cosmosaccount.New(
```

Comment:
Instead of initializing a new one maybe we can use this one here that already initialized: 

https://github.com/tendermint/starport/blob/79799eb/starport/cmd/network.go

For this [newNetwork](https://github.com/tendermint/starport/blob/79799eb/starport/cmd/network.go#L70) needs to return `cosmos.AccountRegistry` alongside the network instance.